apps/api-gateway/src/oid4vc-issuance/dtos/issuer-sessions.dto.ts

```
@@ -0,0 +1,407 @@
+  @ApiProperty({
+    description: 'Unique ID of the supported credential',
+    example: 'DrivingLicenseCredential-mdoc'
+  })
+  @IsString()
+  credentialSupportedId: string;
+
+  @ApiProperty({
+    description: 'Signer options for credential issuance',
+    example: {
+      method: 'x5c',
+      x5c: [
```

ESLint

[error] There should be no linebreak after '['.

(array-bracket-newline)